activerecord/test/cases/instrumentation_test.rb

```diff
@@ -48,7 +48,8 @@ def test_payload_name_on_update_all
     def test_payload_name_on_eager_load
       ActiveRecord::Base.schema_cache.add(Author.table_name)
       notification = capture_notifications("sql.active_record") { Book.eager_load(:author).to_a }
-      assert_equal "Book Eager Load", notification.first.payload[:name]
+        .find { _1.payload[:sql].match?("SELECT") }
```

Comment:
`_1` is not obvious in this case - you need to read the previous code to know what it refers to. I initially thought it is `book`.

```suggestion
        .find { |notification| notification.payload[:sql].match?("SELECT") }
```

Reply:
@fatkodima Thanks for taking a look!

I understand, but this is the same for all the other tests here:
https://github.com/rails/rails/blob/92481359a7f74261059a292a7cea61e68144b57e/activerecord/test/cases/instrumentation_test.rb#L26-L46

Reply:
It's a [core feature](https://docs.ruby-lang.org/en/master/Proc.html#class-Proc-label-Anonymous+block+parameters) to Ruby, added in [2.7](https://github.com/ruby/ruby/blob/v2_7_0/NEWS#label-Numbered+parameters) -- so I think it's safe to use.

I was always taught to adapt to the consistency of the code in the file or project you're editting, but I understand that syntax sugar maybe confusing.

I'm happy to make it easier to read though if you feel strongly about it. :bow: